cobigen-maven/cobigen-maven-systemtest/src/test/resources/testdata/systemtest/Oasp4JTemplateTest/TestDifferentFileSystems/pom.xml

```diff
@@ -69,7 +67,7 @@
           <dependency>
             <groupId>com.devonfw.cobigen</groupId>
             <artifactId>tsplugin</artifactId>
-            <version>2.4.1</version>
+            <version>2.5.0-SNAPSHOT</version>
```

Comment:
I think it is better not to use SNAPSHOT versions, does version `2.4.1` fail for you?

Reply:
2.4.1 is working too, I've just tested it.

Reply:
Perfect. Then please change it back to 2.4.1. The problem with SNAPSHOT versions is that they are installed only on your local computer. I would not have the same version as you if I run this pom.